less/paging.less

```diff
@@ -0,0 +1,65 @@
+// ID-132
+.id7-paging {
+  .container();
+  padding-top: 0;
+  padding-bottom: 0;
+}
+
+.id7-paging,
+.id7-main-content {
```

Comment:
I _think_ it needs a bit of specificity help because of the old `a:not(.btn):link` thing. Might be able to lighten it a bit...